cmd/root.go

```diff
@@ -188,7 +186,11 @@ func AddCommands(cmd *cobra.Command) {
 	cmd.PersistentFlags().Uint32Var(&config.Beacon.BeaconSyncNumBallots, "beacon-sync-num-blocks",
 		config.Beacon.BeaconSyncNumBallots, "Numbers of blocks to wait before determining beacon values from them.")
 
-	/**======================== Post Flags ========================== **/
+	/**======================== Tortoise Flags ========================== **/
+	cmd.PersistentFlags().Uint32Var(&config.Tortoise.Hdist, "tortoise-hdist",
```

Comment:
i found in spacecraft configs use just "hdist"

Reply:
i forgot to update it, we are using default 10 anyways